cmd/api/src/api/v2/cypherquery.go

```
@@ -45,13 +43,25 @@ type CypherQueryPayload struct {
 
 // Helper function to handle error conditions in CypherQuery.
 func handleCypherDBErrors(response http.ResponseWriter, request *http.Request, err error) {
+	var errorResp *api.ErrorWrapper
+	errorCategoryLabel := cypherQueryErrorTypeUnknown
```

golangci-lint (2.12.2)

[major] ineffectual assignment to errorCategoryLabel

(ineffassign)